src/Tree/NodeWrapper.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TransitionGroup } from 'react-transition-group';
+
+export default class NodeWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      enableTransitions: props.transitionDuration > 0,
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({
```

Comment:
Think we can save ourselves excessive state updates here by checking `nextProps.transitionDuration !== this.props.transitionDuration`.

Correct me if I'm missing something though.

Reply:
Yes, I overlooked this. The fix is pushed.